tools/gen_openvino_dockerfile.py

```diff
@@ -123,37 +123,19 @@ def dockerfile_for_windows(output_file):
     df += """
 SHELL ["cmd", "/S", "/C"]
 
-# Build instructions:
-# https://github.com/openvinotoolkit/openvino/wiki/BuildingForWindows
+# instructions:
+# https://docs.openvino.ai/2023.3/openvino_docs_install_guides_installing_openvino_from_archive_windows.html
```

Comment:
Can you make a note that we are building from source for linux but installing from the openvino repository for windows?